docs/README.md

```diff
@@ -36,19 +36,52 @@ As per the [Team Data Science Process (TDSP)](https://docs.microsoft.com/en-us/a
 
 An important design goal is modularity: the components provide utility on their own, and can be re-used in different scenarios to solve other problems at a lower incremental cost.
 
-# Scenario (business understanding)
+# Scenario
 
-TBD
+The solution deals with an imaginary IoT-enabled manufacturing environment comprised of generalized rotational equipment, which may include pumps, turbines, gearboxes, compressors, and engines. 
 
-# Data acquisition and understanding
+The machines are equipped with sensors from which telemetry is regularly transmitted to the cloud in near real time. Maintenance logs are also available and, among other things, contain records of failure events indicating exact points in time when a machine had a critical failure of a particular type.
```

Comment:
The machines are equipped with sensors that transmit telemetry to the cloud in near real time. Maintenance logs contain record the timestamp of each failure and labels of which component(s) failed.

Reply:
This statement doesn't match our data.

Reply:
The second sentence, I mean.

Reply:
Understood.  I may have fallen back to my experience from my team with AWS where all we cared about was which component needed to be replaced.  How big is the difference between "critical failure of a particular type" and "labels of which component(s) failed"?  Would the second sentence match our data better if it said "Maintenance logs contain record the timestamp of each failure and critical failure of a particular type"?  What else is encapsulated within "among other things"?